c++/src/capnp/rpc.c++

```diff
@@ -233,6 +234,13 @@ public:
     return (id & REVERSE_ALLOCATED_BIT<Id>) != 0;
   }
 
+  bool isRealtime(Id id) {
+    // Returns whether this ID represents a realtime message.
+    // Realtime messages have BOTH the One-Way bit and the Reverse-Allocated bit set.
```

Comment:
This is a little bit confusing framing, as realtime actually has nothing to do with reverse allocation, but we're reusing the bit since there's no such thing as a one-way reverse-allocated message. (Note that `isReverseAllocated()` certainly shouldn't return true for realtime messages!)

I think probably we should define four values like:

```
enum QuestionIdType {
  NORMAL = 0,
  REVERSE_ALLOCATED = 1,
  PIPELINE_ONLY = 2,
  REALTIME = 3
}
```

And then we expect the top two bits to match one of these four.